### DISCUSSION TITLE
Raw files

### DISCUSSION THREAD

**Discussion by Mskater** [deepmodeling/deepmd-kit#672] [Q&A]
Can anyone please tell how one obtains the initial data content to make the .raw files?
I cannot find the beginning of this preparation in the docs. 
I'm a new grad student. 
Thanks

**Comment by njzjz** [comment]
You need to do some _ab initio_ calculations.

**Reply by Mskater** [reply]
OK, thanks.
But specifically, when an OUTCAR file from DFT is converted via dpdata program to DeePMD files, are the raw files created automatically in proper format, or do I have to do something extra?
  Also, how to specify the set(s) to be trained into the input file?  I don't see how to include them in input files?  I'm trying to learn how to train and test the Silicon files.
Thanks very much.

**Reply by njzjz** [reply]
DeePMD-kit only needs numpy binary files. The raw files are used to convert to numpy files by numpy. You are free to directly create numpy files using dpdata or others.

You can include your data here
https://github.com/deepmodeling/deepmd-kit/blob/773ed37ce4f95d04ce13ca3a1348fe4295f9d7d8/examples/water/train/water_se_a.json#L45
